packages/opentelemetry-api/src/trace/instrumentation/Instrumentation.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MeterProvider } from '../..';
+import { Logger } from '../../common/Logger';
+import { TracerProvider } from '../tracer_provider';
+
+/** Interface Instrumentation to apply patch. */
+export interface Instrumentation {
```

Comment:
I would be in favor of adding them in the base instrumentation since they are not useful as-is inside the API

Reply:
we are keeping current plugin in api, so I put it in the same place so once all plugins are refactored this can be removed. But they can also be kept in instrumentation package, I'm fine for both.

Reply:
I think I would prefer them in the instrumentation package